project/ReflectiveCodeGen.scala

```diff
@@ -49,6 +52,19 @@ object ReflectiveCodeGen extends AutoPlugin {
           }.value,
         // HACK: make the targets mutable, so we can fill them while running the above PB.generate
         PB.targets := scala.collection.mutable.ListBuffer.empty,
+        // Put an artifact resolver that returns the project's classpath for our generators
+        PB.artifactResolver := Def.taskDyn {
+          val cp = (fullClasspath in Compile in ProjectRef(file("."), "play-grpc-generators")).value.map(_.data)
+          val oldResolver = PB.artifactResolver.value
+          Def.task { (artifact: BridgeArtifact) =>
+            artifact.groupId match {
+              case "com.lightbend.akka.grpc" =>
+                cp
+              case _ =>
+                oldResolver(artifact)
+            }
+          }
+        }.value,
```

Comment:
This is the most relevant change since without it the protoc compiler no longer can locate our code and fails.